components/intl_provider/intl_provider.tsx

```diff
@@ -7,7 +7,7 @@ import {IntlProvider as BaseIntlProvider} from 'react-intl';
 import {MessageFormatElement} from '@formatjs/icu-messageformat-parser';
 
 import {Client4} from 'mattermost-redux/client';
-import {setLocalizeFunction} from 'mattermost-redux/utils/i18n_utils';
+import {setLocalizeFunction} from '@mattermost/components/utils/i18n';
```

Comment:
Created a `utils` subfolder inside `components` and moved some of the needed mattermost-redux functions there